src/context/CapturesContext.js

```diff
@@ -31,19 +39,43 @@ export const CapturesContext = createContext({
 
 export function CapturesProvider(props) {
   const { orgId, orgList } = useContext(AppContext);
+  const { searchParams } = props;
+
+  const {
+    rowsPerPage: rowsPerPageParam = undefined,
+    page: pageParam = undefined,
+    order: orderParam = undefined,
+    orderBy: orderByParam = undefined,
+    ...filterParams
+  } = Object.fromEntries(searchParams || []);
+
   const [captures, setCaptures] = useState([]);
   const [captureCount, setCaptureCount] = useState(0);
   const [capture, setCapture] = useState({});
-  const [page, setPage] = useState(0);
-  const [rowsPerPage, setRowsPerPage] = useState(25);
-  const [order, setOrder] = useState('desc');
-  const [orderBy, setOrderBy] = useState('created_at');
+  const [page, setPage] = useState(Number(pageParam) || DEFAULT_PAGE);
+  const [rowsPerPage, setRowsPerPage] = useState(
+    Number(rowsPerPageParam) || DEFAULT_ROWS_PER_PAGE
+  );
+  const [order, setOrder] = useState(orderParam || DEFAULT_ORDER);
+  const [orderBy, setOrderBy] = useState(orderByParam || DEFAULT_ORDER_BY);
   const [isLoading, setIsLoading] = useState(false);
   const [filter, setFilter] = useState(
-    new FilterModel({ organization_id: ALL_ORGANIZATIONS })
+    FilterModel.fromSearchParams({
+      organization_id: ALL_ORGANIZATIONS,
+      ...filterParams,
+    })
   );
 
   useEffect(() => {
+    handleQuerySearchParams({
```

Comment:
Does this need to be inside a conditional like in the Verify and Grower contexts?

Reply:
I think it's the others that are incorrect.
I suspect the `if (searchParams)` check is an artefact of a previous way of doing this.
We probably shouldn't check it as we're not using it within this `useEffect`. I'll do some more testing.